src/demo-app/datepicker/datepicker-demo.ts

```diff
@@ -31,7 +31,7 @@ export class DatepickerDemo {
   lastDateInput: Date | null;
   lastDateChange: Date | null;
 
-  dateFilter = (date: Date) => date.getMonth() % 2 == 1 && date.getDate() % 2 == 0;
+  dateFilter = (date: Date) => date.getFullYear() % 2 == 0 && date.getMonth() % 2 == 1 && date.getDate() % 2 == 0;
```

Comment:
Could be simplified to `!date.getFullYear() && date.getMonth() % 2 == 1 && !date.getDate() % 2;`

Reply:
Assume you meant: `!date.getFullYear() % 2 && date.getMonth() % 2 && !date.getDate() % 2;`?

Reply:
Ups, yes. Bad copy/paste :D